server/shared-models/src/main/scala/com/prisma/shared/models/ProjectJsonFormatter.scala

```diff
@@ -163,7 +165,38 @@ object ProjectJsonFormatter {
     private def addDiscriminator(json: JsObject, fn: Function) = json ++ Json.obj(discriminatorField -> fn.typeCode.toString)
   }
 
-  implicit lazy val relation                  = Json.format[Relation]
+  implicit val relationWrites: Writes[Relation] = (
```

Comment:
I think we should remove the implicit keyword for the Writes and Reads. It is enough that the Format is marked implicit.

Reply:
Done